vendor/assets/javascripts/brainstem/brainstem-model.coffee

```diff
@@ -95,6 +105,16 @@ class window.Brainstem.Model extends Backbone.Model
     else
       super(field)
 
+  set: (key, val, options) ->
+    super
+
+    _.extend(@unsavedAttributes || {}, @changed)
```

Comment:
It might be more informative to use an `if @unsavedAttributes?`

Aside: Can we add spec coverage to ensure someone doesn't switch the lines in the constructor? Or would it break other specs already?